### PR TITLE
Meet mentor

### PR DESCRIPTION
**Pull request by steelerose**
We created models for concept tags ('Tag') before realizing that those applied to a later sprint (started working from the wireframes instead of the issue description).  Those have been left in the branch for later use, but haven't been implemented with any views or controllers

### CODE REVIEW

File: lib/user.rb [hunk 1]
```diff
@@ -145,6 +145,22 @@ def about_me
     end
   end
 
+  def claimed_requests
+    if is_mentor?
+      Request.where(mentor_uuid: self.uuid)
+    else
+      Request.where(member_uuid: self.uuid) - Request.where(member_uuid: self.uuid, mentor_uuid: nil)
```
Comment (ErinCall) [inline lib/user.rb]: Will you rewrite this so it's a single trip to the database?

``` Ruby
Request.where(["member_uuid = ? and mentor_uuid is not null", uuid])
```

File: app/models/request.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Request < ActiveRecord::Base
+  has_many :tags, as: :tagable
+  has_many :concepts, through: :tags
+  validates :content, presence: true
+  validates :title, presence: true, length: { maximum: 100 }
+  validates :contact_info, presence: true
+end
```
Comment (ErinCall) [inline app/models/request.rb]: Please rename this model to something like MeetingRequest so it's not easily confused with the rails `request` object.

Comment (steelerose) [inline app/models/request.rb]: Changes have been committed and pushed to github.

**Comment by steelerose** [comment]
"Adds markdown to concept descriptions" should also be fixed now.

**Comment by steelerose** [comment]
Added another pull request -

The meet mentor stuff has been cleaned up (html/css) to look more like the
wireframes.

**Comment by steelerose** [comment]
"Clean up Concepts" is updated and ready to go.

**Comment by steelerose** [comment]
Steven, Michael, and I all looked at the Travis CI error for the "adds
markdowns for concepts" pull request, and we can't figure it out - all of
the specs on our computer are running fine.

I pulled from upstream (since you merged some of the pull requests today)
and resolved conflicts, and we're hoping that something went wonky last
time, but magically works this time.

Updated and pushed - waiting to see if it all tests run green on GitHub.

**Comment by steelerose** [comment]
We just made a pull request for the "projects" issue.

**Pull request merged**